pkg/server/container_create.go

```diff
@@ -214,6 +214,8 @@ func (c *criContainerdService) generateContainerSpec(id string, sandboxPid uint3
 		// TODO(random-liu): [P1] Set selinux options.
 
 		// TODO(random-liu): [P2] Add apparmor and seccomp.
+
+		g.SetProcessNoNewPrivileges(securityContext.GetNoNewPrivs())
```

Comment:
Please add unit test for this. You could take the existing unit test as a reference. :)